common/utils/src/main/resources/error/error-classes.json

```diff
@@ -1272,6 +1278,12 @@
     ],
     "sqlState" : "42710"
   },
+  "FIELD_INDEX_ON_ROW_WITHOUT_SCHEMA" : {
+    "message" : [
+      "fieldIndex on a Row without schema is undefined."
+    ],
+    "sqlState" : "42000"
```

Comment:
How about to assign some unique number in the range `42XXX`.

Reply:
Sounds good. I think we can use `42601` - A character, token, or clause is invalid or missing -